Add PATH_SUFFIXES to look inside the sub-directory `vlc` - the original statement
only check for `vlc.h` inside the directory `include`, `bin`, etc.
In Guix, however, `vlc.h` lies inside `/gnu/store/<hash>-vlc-x.y.z/include/vlc/vlc.h`

--- a/cmake/modules/FindLIBVLC.cmake
+++ b/cmake/modules/FindLIBVLC.cmake
@@ -29,7 +29,7 @@ FIND_PATH(LIBVLC_INCLUDE_DIR vlc/vlc.h
   c:/msys/local/include
   NO_DEFAULT_PATH
   )
-FIND_PATH(LIBVLC_INCLUDE_DIR vlc.h)
+FIND_PATH(LIBVLC_INCLUDE_DIR vlc.h PATH_SUFFIXES vlc)
 
 #Put here path to custom location
 #example: /home/user/vlc/lib etc..
